### DISCUSSION TITLE
Is it possible to develop an arm version?

### DISCUSSION THREAD

**Discussion by yangtao199** [Stirling-Tools/Stirling-PDF#726] [General]
在中国其实很多电脑以及服务器在逐渐国产化，国产化的电脑大部分都是arm架构的机器，这就导致arm架构的软件在中国慢慢的大受欢迎。并且现在随着树莓派兴起，好多玩家都想拥有自己的树莓派服务器，这也促使arm架构的软件越来越多。我就是因为有自己的树莓派服务器 但是发现没有arm版本的软件而难过

**Comment by Frooodle** [comment]
![image](https://github.com/Stirling-Tools/Stirling-PDF/assets/77850077/de97caae-4fcd-4f81-beb5-14ee3f70b240)
There is a ARM version
Note it is 64bit only

**Reply by Frooodle** [reply]
If you have issues on your 64 bit machine it could be related to installing wrong 32bit OS 
[Others ](https://github.com/Stirling-Tools/Stirling-PDF/issues/168#issuecomment-1550525756)
